main.py

```diff
@@ -1,2 +1,3 @@
 # list of the alphabet (a-z)
 alphabet = [ chr(ord("a") + i) for i in range(26) ]
+reValphabet = alphabet[::-1] 
```

Comment:
```suggestion
reversed_alphabet = alphabet[::-1]
```
Suggestion based PEP8 best practice naming convention. See [Naming Variables](https://realpython.com/python-pep8/#naming-conventions) in the PEP8 guide.